### ISSUE TITLE
Error Message: local variable 'listVersion' referenced before assignment

### ISSUE THREAD

**Issue by BI0SER** [ThioJoe/YT-Spammer-Purge#639]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I activate the program but this message is displayed

Loading YT Spammer Purge @ 2.15.2...
Checking for updates to program and spam lists...
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1611, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 170, in main
  File "Scripts\files.py", line 715, in get_list_file_version
UnboundLocalError: local variable 'listVersion' referenced before assignment
------------------------------------------------
Error Message: local variable 'listVersion' referenced before assignment

Error - Code: X-2 occurred. This is almost definitely my fault and requires patching (big bruh moment)
Please post a bug report on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report
    (In the mean time, try using a previous release of the program.)

 Press Enter to Exit...

### Release version

2.15.2

### Steps to reproduce

1. Open the program.
2. Error message comes up.
3. (Note) I Just installed the exe file but I followed your steps.

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Loading YT Spammer Purge @ 2.15.2...
Checking for updates to program and spam lists...
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1611, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 170, in main
  File "Scripts\files.py", line 715, in get_list_file_version
UnboundLocalError: local variable 'listVersion' referenced before assignment
------------------------------------------------
Error Message: local variable 'listVersion' referenced before assignment

Error - Code: X-2 occurred. This is almost definitely my fault and requires patching (big bruh moment)
Please post a bug report on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report
    (In the mean time, try using a previous release of the program.)

 Press Enter to Exit...
```

### Screenshots

![image](https://user-images.githubusercontent.com/97829410/153743862-ad908c9d-60b9-4fe4-bfe2-2148f04656ca.png)

**Comment by ThioJoe** [comment]
Huh that's odd, it suggests that there was no list version found in the spam list files, even though they're there.

Can you check if you see the the text files under SpamPurge_Resources/Spam_Lists? Do the lists have a line near the top that says something like `#version=[2022.2.6]` ?

Either way should be fixed with https://github.com/ThioJoe/YT-Spammer-Purge/commit/e2e51669aa88960238cb60e504cdc01633c3dd17 for the next release

**Comment by ThioJoe** [comment]
Ok the crash _should_ be fixed with the latest release, 2.15.3. Still not fully sure lead to that particular problem in the first place though, so if you can provide the info from my previous comment, that would still be helpful.

Let me know if it keeps happening though and I'll re-open this.
